### PR TITLE
Major optimizations

### PR DESCRIPTION
**Pull request by k4nar**
This is not much but as this we be called on every requests we should try to avoid as much overhead as we can.

### CODE REVIEW

File: tests/readonly_app/models.py [hunk 1]
```diff
@@ -16,3 +16,15 @@ def __str__(self):
 
     class ReadOnlyMeta:
         read_only = ["manufacturer"]
+
+
+@python_2_unicode_compatible
+class Book(models.Model):
```
Comment (zebuline) [inline tests/readonly_app/models.py:22]: you should add some tests with this model (write operations)

Comment (k4nar) [reply tests/readonly_app/models.py:22]: Oh yeah that was temporary :) . Sorry about that.

**Comment by ewjoachim** [comment]
@k4nar Added a commit that makes things better. The project is still early, so, sorry about the mess :D LGTY ?

**Comment by zebuline** [comment]
Yeah, lgtm :)

**Pull request merged**